### ISSUE TITLE
Benefit text + Examples Understanding 3.2.6 Consistent Help not the right fit anymore?!

### ISSUE THREAD

**Issue by jake-abma** [w3c/wcag#2414]
In the Understanding doc there is explained what kind of help is good/better/best
https://w3c.github.io/wcag/understanding/consistent-help.html#benefits

This was once applicable when the SC was in its early stage.
As it now only is all about the consistent location, the text should be adjusted to match that situation.

**Comment by jake-abma** [comment]
Same for the Examples, which do not reflect the requirement of the SC, but talks about proper ways of help/contact 

https://w3c.github.io/wcag/understanding/consistent-help.html#examples

**Comment by patrickhlauke** [comment]
related old discussion https://github.com/w3c/wcag/issues/1881
